src/ida_pro_mcp/ida_tools.py

```diff
@@ -75,165 +72,14 @@ def jsonrpc(func: Callable) -> Callable:
     global rpc_registry
     return rpc_registry.register(func)
 
-class JSONRPCRequestHandler(http.server.BaseHTTPRequestHandler):

```

Comment:
this JSON-RPC code moves into a separate IDA Pro plugin file, since its not used in an idalib-based setup.